pyproject.toml

```diff
@@ -19,6 +19,11 @@ classifiers = [
     "Development Status :: 4 - Beta",
     "Intended Audience :: Developers",
     "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
```

Comment:
Poetry [adds these automatically](https://python-poetry.org/docs/pyproject/#classifiers). The Python version classifiers can be completely removed.

```suggestion
```

Reply:
Thank you! I had no idea.